pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -1489,8 +1486,9 @@ func reportGatewayStatus(
 
 	// Setup initial conditions to the success state. If we encounter errors, we will update this.
 	// We have two status
-	// Accepted: did we get *some* data to the data plane
-	// Programmed: did we get *all* the data to the data plane
+	// Accepted: is the configuration valid. We only have errors in listeners, and the status is not supposed to
+	// be tied to listeners, so this is always accepted
+	// Programmed: is the data plane "ready" (note: eventually consistent)
```

Comment:
Would `Ready` make more sense?

Reply:
I tried. Not our decision... see https://github.com/kubernetes-sigs/gateway-api/issues/1832 and https://github.com/kubernetes-sigs/gateway-api/issues/1877

Reply:
APi decided Ready must mean something that ~cannot be implemented (not eventually consistent); since no one implements it it will be deprecated.

Reply:
Ok glad it is not just me. :)